rickynyairo/StackOverflow-lite
# Test Endpoints of various resources

#### What does this PR do?
Generates tests for various endpoints that the API is going to have

#### Description of Task to be completed?
The PR defines the tests that the endpoints have to pass in order to function as required.

#### How should this be manually tested?
Clone the repository [here](https://github.com/rickynyairo/StackOverflow-lite/)

>git clone https://github.com/rickynyairo/StackOverflow-lite/

Navigate to the API directory
install the requirements (this should be done in a virtual environment)

> pip install requirements.txt 

#### Any background context you want to provide?
The API defines a set of endpoints for the StackOverflow-Lite project
 
#### What are the relevant pivotal tracker stories?
[#159747662](https://www.pivotaltracker.com/story/show/159747662)
